src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "role_name")
+    @Enumerated(EnumType.STRING)
+    private RoleName roleName;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public RoleName getRoleName() {
+        return roleName;
+    }
+
+    public void setRoleName(RoleName roleName) {
+        this.roleName = roleName;
+    }
+
+    @Override
+    public String toString() {
+        return "Role{"
+                + "id=" + id
+                + ", roleName=" + roleName
+                + '}';
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
```

Comment:
if it works than ok, but I would use auto generated by intellij

Reply:
try to fix some bugs so changed everything that I can )))

Reply:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.1:check (default) on project spring-security-part-2: Failed during checkstyle configuration: Exception was thrown while processing /Users/oleksii/Documents/GitHub/jv-spring-security-part-2/src/main/java/mate/academy/spring/model/Role.java: IllegalStateException occurred while parsing file /Users/oleksii/Documents/GitHub/jv-spring-security-part-2/src/main/java/mate/academy/spring/model/Role.java. /Users/oleksii/Documents/GitHub/jv-spring-security-part-2/src/main/java/mate/academy/spring/model/Role.java:52:33: expecting RPAREN, found 'role' -> [Help 1]
I have this exception if I change equals method to default. I don't understand why ..

Reply:
Exception for this implementation: ```
@Override
    public boolean equals(Object o) {
        if (this == o) {
            return true;
        }
        if (o instanceof Role role) {
            return Objects.equals(getId(), role.getId()) && getRoleName() == role.getRoleName();
        }
        return false;
    }
```

Reply:
So I have to live previous implementation